lib/DueDatePicker/index.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DayPicker from 'react-day-picker';
+import Overlay from 'react-bootstrap/lib/Overlay';
+import Popover from 'react-bootstrap/lib/Popover';
+import moment from 'moment';
+import cx from 'classnames';
+import Button from '../Button';
+import DueDateHeader from './DueDateHeader';
+import DueDateButton from './DueDateButton';
+
+class DueDatePicker extends Component {
+  constructor(props) {
+    super(props);
+    this.handleDayClick = this.handleDayClick.bind(this);
```

Comment:
We've started using arrow functions rather than binding this in the constructor 😄 

Try it out and see how you think it looks etc